vendor/bat-native-ledger/src/bat/ledger/internal/state/state_util.h

```diff
@@ -0,0 +1,54 @@
+	/* Copyright (c) 2020 The Brave Authors. All rights reserved.
```

Comment:
Nit: Looks like this got indented by mistake.

Reply:
fixed